### ISSUE TITLE
[BUG] - SpellCheck can't find the correct word

### ISSUE THREAD

**Issue by TheUnsleepingAlchemist** [mwittrien/BetterDiscordAddons#2545]
**Describe the bug**
SpellCheck can't recognize words that contain Polish characters, even if the word is correct it still insists that the provided word is incorrect

**To Reproduce**
- Set dictionary to PL
- Type "moze" or "może"

**Expected behavior**
SpellCheck should recognize the incorrect word "moze" and provide "może" instead, if the word is correct it still marks it as an incorrect one

**Screenshots**
![spellcheck01](https://github.com/mwittrien/BetterDiscordAddons/assets/46828559/fcf9e2a1-2004-4b44-917c-f393f81f0571)
![spellcheck02](https://github.com/mwittrien/BetterDiscordAddons/assets/46828559/6bb1ee41-dbbe-453c-84ca-2812b492ba11)

**Discord Build (stable, ptb or canary):**
stable

**Additional context**

**Comment by mwittrien** [comment]
it doesnt mark "może" as incorrect for me
![image](https://github.com/mwittrien/BetterDiscordAddons/assets/23700969/ede4d8b8-6d0f-44a3-bd63-d214c0ddc3a9)

only moze
![image](https://github.com/mwittrien/BetterDiscordAddons/assets/23700969/72190766-2b9b-4499-8b2d-f7c3994f8c50)

**Comment by TheUnsleepingAlchemist** [comment]
I took a quick look at the downloaded plugin to see what could possibly be wrong and my local version was looking like that.
![spellcheck03](https://github.com/mwittrien/BetterDiscordAddons/assets/46828559/affe9d9b-b82a-4860-88fb-f10a3296cf3b)
I tried overwriting it with proper encoding but the problem still existed, should I consider it as a recent discord update bug?
I also tried reinstalling discord, betterdiscord and removing all plugins, this problem seems kinda odd because discord can recognize Polish characters on its own.
